CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required (VERSION 2.8.12)
 
+# Tell cmake 3.0+ that it's safe to clear the PROJECT_VERSION variable in the
+# call to project() below.
+if(POLICY CMP0048)
+  cmake_policy(SET CMP0048 NEW)
+endif()
+
 project (benchmark)
 
 foreach(p
```

Comment:
I think that's not true in general, but it's specifically true for this policy, because it affects the way project() behaves. Either way, doing with the loop for consistency seems fine.